src/librustc_mir/transform/qualify_min_const_fn.rs

```diff
@@ -383,9 +383,14 @@ fn check_terminator(
                         .into(),
                     ));
                 }
+
+                // HACK: This is to "unstabilize" the stabilization of the `transmute` intrinsic
```

Comment:
```suggestion
                // HACK: This is to "unstabilize" the `transmute` intrinsic
```
I don't know what it means to unstabilize a stabilization. ;)

Reply:
that's why it's in quotes. I was hoping the rest of the text would make it clear.

Reply:
I mean it's clear what you mean, just like it's mostly still clear what you mean when there's a typo. I think with my edit/"typo" fix it's just as clear but sounds more like English. ;)